content/join.hbs

```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Spring 2021</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Fall 2021</span> semester</h1>
 			<div class="columns">
 				<div class="column is-half side-by-side current-semester">
 					<h2>Join as a <span class="semester-text">Georgia Tech</span> student</h1>
-					<p>To enroll for Spring 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
+					<p>To enroll for Fall 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>

```

Comment:
Do we have a new schedule document for Fall 2021?

Reply:
Same as below. I will check again